pkg/config/config.go

```diff
@@ -9,16 +9,19 @@ import (
 )
 
 const (
-	FALCO_EBPF_ENGINE  = "falco"
-	CILIUM_EBPF_ENGINE = "cilium"
+	FALCO_EBPF_ENGINE      = "falco"
```

Comment:
```suggestion
	EBPFEngineFalco      = "falco"
```
[Required]: Please follow the [Go convention for naming variables](https://google.github.io/styleguide/go/decisions#constant-names).